### DISCUSSION TITLE
Get Learning Rate(s) Used in Model Training with Auto Regression

### DISCUSSION THREAD

**Discussion by skumarlabs** [ourownstory/neural_prophet#1389] [Q&A - get help using NeuralProphet] [answered]
Hi Team, I am using below config to train my time series model. The guide suggests that leaving `learning_rate` to `None` will automatically find the suitable value. Is there any way I can get the learning rate(s) used to train my model?

```
hyperparams = {
    "epochs": 7,
#     "batch_size": 128,
    "n_lags": 14,
    "n_forecasts": 7,
    "trend_global_local": "local",
    "season_global_local": "local",
}
m = NeuralProphet(**hyperparams)
```

**Comment by leoniewgnr** [comment, marked as answer]
`m.model.learning_rate`

**Reply by skumarlabs** [reply]
Thank you. This helped.
